pkg/cmd/cli/cmd/build.go

```diff
@@ -215,5 +215,6 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().StringVar(&(networkMode), "network", "", "Specify the default Docker Network name to be used in build process")
 	buildCmd.Flags().StringVarP(&(cfg.AsDockerfile), "as-dockerfile", "", "", "EXPERIMENTAL: Output a Dockerfile to this path instead of building a new image")
 	buildCmd.Flags().BoolVarP(&(cfg.KeepSymlinks), "keep-symlinks", "", false, "When using '--copy', copy symlinks as symlinks. Default behavior is to follow symlinks and copy files by content")
+	buildCmd.Flags().StringArrayVar(&cfg.AddHost, "add-host", []string{}, "Specify additional entry added to /etc/hosts in container, multiple --add-host can be used to add multiple entry")
```

Comment:
grammar.  should be:
"Specify additional entries to add to the /etc/hosts in the assemble container, multiple --add-host arguments can be used to add multiple entries"